src/multi_vector_simulator/C1_verification.py

```diff
@@ -100,8 +104,8 @@ def lookup_file(file_path, name):
     """
     if os.path.isfile(file_path) is False:
         msg = (
-            f"Missing file! The timeseries file '{file_path}' \nof asset "
-            + f"{name} can not be found. Operation terminated."
+                f"Missing file! The timeseries file '{file_path}' \nof asset "
+                + f"{name} can not be found. Operation terminated."
```

Comment:
I suppose you did not do this manually? Can you make sure that the same `black` version is installed? Between versions, there may be different formatting. If the online pytest for black does not pass, that may be the reason.

Reply:
I checked with PF yesterday and we have the same version (19.10b0), but I will try and format it again.